packer/packer.json

```diff
@@ -47,6 +47,24 @@
         "kubernetes_version": "{{user `kubernetes_version`}}",
         "kubernetes_cni_version": "{{user `kubernetes_cni_version`}}"
       }
+    },
+    {
+      "name": "gcp-ubuntu-16.04",
+      "type": "googlecompute",
+      "project_id": "{{user `project_id`}}",
+      "source_image": "ubuntu-1604-xenial-v20180306",
```

Comment:
I think we should always be passing these in via a variable/vars file.

Reply:
They're not specific to a region, so I think it makes sense to have them written in here. No reason to add a bunch of extra arguments that'll always be the same

Reply:
the intention here is that users can supply them should they have a base image that they would like to use. i think it is fine to have a default that we use, but also thinks this needs to be overridable.